docs/tools/vdb_table/data/activeloop.json

```
@@ -33,9 +33,9 @@
     "comment": ""
   },
   "hybrid_search": {
-    "support": "",
-    "source_url": "https://docs.activeloop.ai/performance-features/querying-datasets/query-syntax",
-    "comment": "While you can run embedding search + contains(text, 'keywoard') or multiple those (keyword search inside text tensor) since BM25 not available I wouldn't call a full hybrid search.  https://docs.activeloop.ai/performance-features/querying-datasets/query-syntax"
+    "support": "full",
+    "source_url": "https://docs.deeplake.ai/4.0/guide/rag/",
```

I was redirected with the current url because the version is out of date. I suggest using the `latest` prefix instead.
```suggestion
    "source_url": "https://docs.deeplake.ai/latest/guide/rag/",
```